py/helpers/cache/tests/test_cache_local.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+
+from fvs.tests.colorful_tests import ColorfulTestRunner
+from helpers.cache.cache_factory import LocalCache
+from helpers.cache.cache_factory import RedisCache
+from helpers.cache.cache_helper import CacheKeyMaker
+
+
+class CacheLocalTest(unittest.TestCase):
+
+  @staticmethod
+  def test_local_cache():
+    # Initialize the cache
+    cache = LocalCache(
+        config={
+            "local_base_dir": ".cache_local_data",
```

Comment:
Hard to know what the working directory is in a github action. Maybe use '/tmp/.cache_test_dir_{unique_code}'

Reply:
this config can easily be changed as per use case , and I guess it should create the `.cache_local_data` folder in github action also , if not will change it accordingly